cdc/sink/mysql/mysql_worker.go

```diff
@@ -112,9 +114,15 @@ func (w *mysqlSinkWorker) run(ctx context.Context) (err error) {
 	}()
 
 	flushRows := func() error {
-		if len(toExecRows) == 0 {
+		if rowEvtQueue.Empty() {
 			return nil
 		}
+
+		toExecRows, ok := rowEvtQueue.DequeueAll()
```

Comment:
Will there be a large memory copy here?

Reply:
Yes, the copy can be avoided. The only reason it exists here is sink processes the dequeued events as a slice in the previous code.